lib/osf-components/addon/components/file-metadata-manager/component.ts

```diff
@@ -30,7 +34,9 @@ export interface FileMetadataManager {
     metadata: CustomFileMetadataRecordModel;
     targetMetadata: CustomItemMetadataRecordModel;
     file: FileModel;
-    target: (AbstractNodeModel);
+    target: (NodeModel | RegistrationModel);
+    targetInstitutions: InstitutionModel[];
```

Comment:
I would singularize this but that is personal preference.

Reply:
We generally pluralize arrays.